src/libs/brands.ts

```diff
@@ -23,15 +23,43 @@ import terraLogoShadow from 'src/images/brands/terra/logo-wShadow.svg'
 
 const nonBreakingHyphen = '\u2011'
 
+/**
+ * BrandConfiguration
+ * @property name - Brand name.
+ * @property queryName - Used to construct return URLs for FireCloud.
+ * @property welcomeHeader - Landing page header text.
+ * @property description - Landing page text.
+ * @property hostName - Host name for branded site.
+ * @property docLinks - Links shown on landing page.
+ * @property docLinks[].link - Link URL.
+ * @property docLinks[].text - Link text.
+ * @property logos - URLs for logo images.
+ * @property logos.color - Brand logo.
+ * @property logos.white - Light version of brand logo used against dark backgrounds.
+ * @property catalogDataCollectionsToInclude - Customize which datasets show up in the Data Catalog.
+ */
+export type BrandConfiguration = {
+  name: string
+  queryName: string
+  welcomeHeader: string
+  description: string
+  hostName: string
+  docLinks: { link: string; text: string }[]
+  logos: {
+    color: string
+    white: string
+    [otherLogoType: string]: string
+  }
+  catalogDataCollectionsToInclude?: string[]
+}
 
 /**
  * Configuration for Terra co-brands (a.k.a. white label sites)
  * https://broadworkbench.atlassian.net/wiki/spaces/WOR/pages/2369388553/Cobranding+and+White+Label+Sites
  */
-export const brands = {
+export const brands: { [brandName: string]: BrandConfiguration } = {
   anvil: {
     name: 'AnVIL',
-    signInName: 'AnVIL',
```

Comment:
This was originally used on the sign in page, but was removed in #3587.

https://github.com/DataBiosphere/terra-ui/pull/3587/files#diff-d0656efad62df4ece3d0456e18cdbc72120ed48e9ba8705c649c39c2b088af34L30-R25